sharktank_models/action.yml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: IREE Test Suite Regression Test
+
+inputs:
+  model:
+    description: "The model to run threshold and benchmark tests"
+    required: true
+  sku:
+    description: "Type of SKU to test"
+    required: true
+  backend:
+    description: "Type of backend to use"
+    required: true
+  run_benchmarks:
+    description: "If flag is set to true, benchmark tests will run"
+    type: boolean
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout the code
+      uses: actions/checkout@v2
```

Comment:
Update this to closer to latest and also pin action versions to commit hashes rather than use tags. Tags can (and often are) changed outside our control, weakening our supply chain

https://github.com/iree-org/iree-test-suites/blob/43ffb5e37c3f72f60d91e498f82419c6a559825b/.github/workflows/test_sharktank_models.yml#L40-L41

More info: https://github.com/ossf/scorecard/blob/main/docs/checks.md#pinned-dependencies